bower.json

```diff
@@ -42,7 +42,7 @@
     "px-forms-design": "^2.0.0"
   },
   "resolutions": {
-    "webcomponentsjs": "^0.7.24",
```

Comment:
Please revert this also. This is an incorrect mix of webcomponents js and Polymer 1.x